lib/watir/elements/image.rb

```diff
@@ -16,18 +16,6 @@ def loaded?
       )
     end
 
-    #
-    # Returns the image's width in pixels.
-    #
-    # @return [Integer] width
-    #
-
-    def width
```

Comment:
And again, you are completely correct. I must have screwed something else up when getting this to work and thought that part was necessary. Thanks!